Classes/Notifications/NotificationModelController.swift

```diff
@@ -24,6 +24,14 @@ extension NotificationViewModel {
     }
 }
 
+// sorting Array<InboxDashboardModel> by date for mentioned filter type
+extension Array where Element == InboxDashboardModel {
+    func sortedMentionedByDate(_ predicate: Bool = true, filter: V3IssuesRequest.FilterType) -> [Element] {
```

Comment:
Hm, what's the reason for the `predicate`? When would you want to call this with `false`? That would just no-op, if I understand this correctly 😄

Reply:
I thought it was a good place for [feature toggle](https://en.wikipedia.org/wiki/Feature_toggle). I removed this argument.